http.go

```diff
@@ -55,6 +56,7 @@ type httpRunner struct {
 	cert              []byte
 	key               []byte
 	skipVerify        bool
+	useCookie         *bool
```

Comment:
The setting of httpRunner may not need to be nil, but httpRequest handles nil, so we have matched it.